### PR TITLE
component when paged list is empty

### PR DESCRIPTION
**Pull request by AnneMedvedeva**
`emptyRowComponent ` field in PagedTable so we can pass an empty row that will appear once the list is empty.

Also, I added return undefined so eslint doesn't throw errors.

### CODE REVIEW

File: src/table/pagedTable/pagedTable.tsx [hunk 1]
```diff
@@ -36,13 +36,18 @@ export interface IPagedTableHook<T> {
 // context for our data table
 // can be used by nested components to for a reload of the data, for instance when editing/deleting rows
 
-export const PagedDataContext = React.createContext<IPagedDataContext>({ reloadData: () => {} })
+export const PagedDataContext = React.createContext<IPagedDataContext>({
```
Comment (revmischa) [inline src/table/pagedTable/pagedTable.tsx:39]: maybe you can say ` reloadData: () => undefined`?

File: src/table/pagedTable/pagedTable.tsx [hunk 2]
```diff
@@ -66,17 +71,19 @@ export function PagedTable<T>({
     columnsSpan = 1
   }
 
+  const listIsNotEmpty = React.useMemo(() => rowsToDisplay && rowsToDisplay.length, [rowsToDisplay])
```
Comment (revmischa) [inline src/table/pagedTable/pagedTable.tsx:74]:
```suggestion
  const listIsNotEmpty = React.useMemo(() => rowsToDisplay?.length, [rowsToDisplay])
```
?

File: src/table/pagedTable/pagedTable.tsx [hunk 3]
```diff
@@ -66,17 +71,19 @@ export function PagedTable<T>({
...
+
+  //render manually passed component if the list is empty
+  const renderEmptyComponent = React.useCallback(
+    () => (emptyRowComponent ? emptyRowComponent : <EmptyTableRow colSpan={columnsSpan} rowText={emptyRowText} />),
```
Comment (revmischa) [inline src/table/pagedTable/pagedTable.tsx:78]: should we yell at them if they pass emptyRowComponent and emptyRowText? you can't have both right?

Comment (AnneMedvedeva) [reply src/table/pagedTable/pagedTable.tsx:78]: right now empty component will just override the text,
how can we tell user that they cant use both?

Comment (revmischa) [reply src/table/pagedTable/pagedTable.tsx:78]: could throw an error

File: src/table/pagedTable/pagedTable.tsx [hunk 4]
```diff
@@ -36,13 +36,18 @@ export interface IPagedTableHook<T> {
 // context for our data table
 // can be used by nested components to for a reload of the data, for instance when editing/deleting rows
 
-export const PagedDataContext = React.createContext<IPagedDataContext>({ reloadData: () => {} })
+export const PagedDataContext = React.createContext<IPagedDataContext>({
+  reloadData: () => {
+    return undefined
```
Comment (rkyslyy) [inline src/table/pagedTable/pagedTable.tsx]: Just curious, why can't it be a `void` function?

Comment (AnneMedvedeva) [reply src/table/pagedTable/pagedTable.tsx]: eslint throws an error that it's an unexpected empty func

Comment (rkyslyy) [reply src/table/pagedTable/pagedTable.tsx]: I see

**Pull request merged**